docs/deploy-cephfs.md

```diff
@@ -69,33 +73,34 @@ YAML manifests are located in `deploy/cephfs/kubernetes`.
 **Deploy RBACs for sidecar containers and node plugins:**
 
 ```bash
-$ kubectl create -f csi-attacher-rbac.yaml
-$ kubectl create -f csi-provisioner-rbac.yaml
-$ kubectl create -f csi-nodeplugin-rbac.yaml
+kubectl create -f csi-attacher-rbac.yaml
+kubectl create -f csi-provisioner-rbac.yaml
+kubectl create -f csi-nodeplugin-rbac.yaml
 ```
 
 Those manifests deploy service accounts, cluster roles and cluster role bindings. These are shared for both RBD and CephFS CSI plugins, as they require the same permissions.
 
 **Deploy CSI sidecar containers:**
 
 ```bash
-$ kubectl create -f csi-cephfsplugin-attacher.yaml
-$ kubectl create -f csi-cephfsplugin-provisioner.yaml
+kubectl create -f csi-cephfsplugin-attacher.yaml
+kubectl create -f csi-cephfsplugin-provisioner.yaml
 ```
 
 Deploys stateful sets for external-attacher and external-provisioner sidecar containers for CSI CephFS.
 
 **Deploy CSI CephFS driver:**
 
 ```bash
-$ kubectl create -f csi-cephfsplugin.yaml
+kubectl create -f csi-cephfsplugin.yaml
 ```
 
 Deploys a daemon set with two containers: CSI driver-registrar and the CSI CephFS driver.
 
 ## Verifying the deployment in Kubernetes
 
-After successfuly completing the steps above, you should see output similar to this:
+After successfully completing the steps above, you should see output similar to this:
+
 ```bash
 $ kubectl get all
```

Comment:
need to remove `$` here also

Reply:
No @Madhu-1 . The `$` has to be removed only when there is no output